solr/packaging/test/test_launch_solr_adminconsole.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load bats_helper
+
+setup() {
+  common_clean_setup
+}
+
+teardown() {
+  # save a snapshot of SOLR_HOME for failed tests
+  save_home_on_failure
+
+  solr stop -all >/dev/null 2>&1
+}
+
+@test "assert able to launch solr admin console" {
+  run solr start -c
+
+  # Replace 'YOUR_ADMIN_CONSOLE_URL' with the actual admin console URL
+  run curl -s -o /dev/null -w "%{http_code}" http://localhost:${SOLR_PORT}/solr/
+
+  # Check if the HTTP response code is 200 (OK)
+  assert_output --partial "200"
```

Comment:
The test failed with the bug and passed without it.

Reply:
Is the `--partial` part accurate -- won't `200` be the entirety of the output?

Reply:
I modified the test to instead match exact output.